container/worker/run_openqa_worker.sh

```diff
@@ -1,18 +1,25 @@
 #!/bin/bash
-# shellcheck disable=SC2012
+# shellcheck disable=SC2012,SC2154
 set -e
+set -x
 
 if [[ -z $OPENQA_WORKER_INSTANCE ]]; then
   OPENQA_WORKER_INSTANCE=1
 fi
 
-/root/qemu/kvm-mknod.sh
+if [[ -z $qemu_no_kvm ]] || [[ $qemu_no_kvm -eq 0 ]]; then
+  if [ -e "/dev/kvm" ] && getent group kvm > /dev/null; then
+    /root/qemu/kvm-mknod.sh
 
-if [ -e "/dev/kvm" ] && getent group kvm > /dev/null; then
-  groupmod -g "$(ls -lhn /dev/kvm | cut -d ' ' -f 4)" kvm
-  usermod -a -G kvm _openqa-worker
-else
-  echo "Warning: /dev/kvm doesn't exist. If you want to use KVM, run the container with --device=/dev/kvm"
+    group=$(ls -lhn /dev/kvm | cut -d ' ' -f 4)
+    groupmod -g "$group" --non-unique kvm
+    usermod -a -G kvm _openqa-worker
+  else
+    echo "Warning: /dev/kvm doesn't exist. If you want to use KVM, run the container with --device=/dev/kvm"
+  fi
 fi
 
+qemu-system-x86_64 -S &
```

Comment:
You could try to run it with `-enable-kvm` parameter unless `$qemu_no_kvm` is set.

Reply:
I did in a previous commit, but I understood that was not a good idea (https://github.com/os-autoinst/openQA/pull/3787#discussion_r603274782)